_posts/2022-01-19-7.0.0-release.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Apache Arrow 7.0.0 Release"
+date: "2022-01-19 00:00:00 -0600"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 7.0.0 release. This covers
+over 3 months of development work and includes [**474 resolved issues**][1]
+from [**?? distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 6.0.1 release, Rémi Dattai and Alessandro Molina have been invited to be committers. 
+Daniël Heres and Yibo Cai have joined the Project Management Committee (PMC). 
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+TBD
+
+## Arrow Flight RPC notes
+
+TBD
```

Comment:
```suggestion
The Flight specification has been clarified to note that schemas are expected to be IPC-encapsulated on the wire.

Documentation has been generally improved; see the [Arrow Cookbook](https://arrow.apache.org/cookbook/) for recipes on how to use Flight in Python and R, and a new [example](https://github.com/apache/arrow/blob/master/cpp/examples/arrow/flight_grpc_example.cc) on how to use Flight and gRPC services on the same port.

This release includes Arrow Flight SQL, a protocol for using Arrow Flight to execute queries against and fetch metadata from SQL databases. Support is included for C++ and Java. A more detailed blog post is forthcoming. Note that development is ongoing and the specification is currently experimental.
```

Reply:
CC @jduong @rafael-telles @vfraga if there's anything else you would like to add about Flight SQL here (of course we can have a blog post specifically about Flight SQL as well).

Reply:
Does "support is included for C++" mean _just_ for C++, or has support also been integrated in bindings that use the C++ API, like Python?

Reply:
@chrish42 in this case it means only C++, I'll update this to clarify.